src/main/webapp/script.js

```diff
@@ -0,0 +1,97 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Set the link for the sign-in page
```

Comment:
period@end (or remove from all (just keep it consistent)? I believe tommy and I both use the period@end convention though). Please propagate to other comments, as well.

Reply:
Vidhya told me that the convention is to use period@end.

Reply:
Yup I've been using a period as my convention too, I just forgot here